### ISSUE TITLE
Create and update prototypes for new Wagtail pages for upcoming webinars

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#1787]
**What we're after**: After determining needs and timeline for late 2018 and 2019 educational outreach webinars and conferences  (#1785), we'll use that information to update protoypes designed by the Information Division in order to move those pages off of transition and into Wagtail. Because these pages include registration links, files to download, and images, we'll need to carefully consider existing templates and design needs. (See #1785 for background on the pages.) 

We'll work on webinar pages in this issue since this is what's coming up. Future work on conference pages will be in a future issue.

**Completion criteria**
- [X] Decide which prototypes will be needed. Based on the current site and Information Division wishes, we need prototypes for:
 - [x] Overall page listing upcoming events and linking them
 - [x] Information page for a particular type of webinar
 - [x] Prototype for a page containing materials for webinars
 - [x] Prototype for a page containing day-of-webinar information for attendees

- [x] Update the above prototypes in time for any fall 2018 webinars (or 2019 events) as planned.

**Comment by dorothyyeager** [comment]
We're heading in a good direction and possibly may have pages go live in time for October reporting webinars.

**Comment by JonellaCulmer** [comment]
Please ignore this issue as part of old ticket closeouts. Per message from @dorothyyeager on slack 8/31.

**Comment by dorothyyeager** [comment]
@JonellaCulmer and I reviewed the last page being developed and sent along our recommendations to Info Division today. Once that page is finalized, we are just waiting on the pay.gov site for the webinars to be finalized. Then we'll need to update each page with that link, doublecheck SEO and then we'll be ready to publish!

**Comment by dorothyyeager** [comment]
The new outreach pages for promoting are live!
Trainings is at https://www.fec.gov/help-candidates-and-committees/trainings/
The webinar-specific page for October reporting webinars is at https://www.fec.gov/help-candidates-and-committees/trainings/october-reporting-fecfile-webinars/. Future webinars will have similar targeted pages.
The other-two pages are not yet live, as they relate to the day-of a webinar and are intended only for paid attendees. 
We can close this ticket and move on to the work in #2319
